tahoe_idp/settings/common.py

```diff
@@ -0,0 +1,51 @@
+# fdddlake8: nodddqa: E501
+import warnings
+from django.core.exceptions import ImproperlyConfigured
+
+
+def magiclink_settings(settings):
+    settings.LOGIN_FAILED_REDIRECT = getattr(settings, 'MAGICLINK_LOGIN_FAILED_REDIRECT', '')
+
+    try:
+        settings.TOKEN_LENGTH = int(getattr(settings, 'MAGICLINK_TOKEN_LENGTH', 50))
```

Comment:
Thanks, I think it's helpful to keep the `MAGICLINK_TOKEN_LENGTH` which is helpful to avoid corrupting the namespace.

using the `settings/common.py` is desired but not mandatory.

However, using `settings/common.py` while polluting settings with a lot of `TOKEN_LENGTH` and `AUTH_TIMEOUT` which implies a Django-wide setting is a bit misleading to the engineer using this plugin.

Please follow either of the following options:

 - Revert back to the old `magiclink_settings.py` (I don't prefer it, but it's okay)
 - Add the prefex to the settings `TOKEN_LENGTH` -> `MAGICLINK_TOKEN_LENGTH`